src/rgw/motr/synchronization/rgw_motr_gc_sync.cc

```diff
@@ -0,0 +1,77 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=2 sw=2 expandtab ft=cpp
+
+/*
+ * Ceph - scalable distributed file system
+ *
+ * SAL implementation for the CORTX Motr backend
+ *
+ * Copyright (C) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ *
+ */
+
+#include "motr/synchronization/rgw_motr_gc_sync.h"
+
+int MotrGCThreadSync::lock(const std::string &lock_name, MotrLockType lock_type, utime_t lock_duration, const std::string locker_id) {
```

Comment:
This is going beyond 80 char limit

Reply:
@jjxsg : Ceph's code is not following 80 char limit. See here: src/cls/lock/cls_lock_client.cc, line "int Lock::break_lock(IoCtx *ioctx, const std::string& oid, const entity_name_t& locker)"

Reply:
Hi, @DPG17, yes ceph code doesn't follow the 80 char limit, but the google c++ coding standard suggests to use max 80 char.
code becomes hard to read and you have to scroll horizontally to see the code, which is not preferable.
https://google.github.io/styleguide/cppguide.html

Reply:
80 char limit in source code is done

Reply:
We are borrowing the logic from CEPH. Code and coding standards we need to follow our own.

Reply:
@jjxsg : We are writing code for Ceph, just like other open source developers for Ceph. We've to follow Ceph's coding style.

Reply:
I believe we are writing code for Cortx and we are using CEPH's infrastructure to enable it.